setup.cfg

```diff
@@ -0,0 +1,60 @@
+[bdist_wheel]
+universal = 1
+
+
+[tool:pytest]
+norecursedirs = .* build dist *.egg tmp*
+testpaths = tests pytest_localstack
+timeout = 300
+addopts =
+	-W error
+
+
+[flake8]
+exclude =
+	.*/
+	__pycache__
+	setup.py
+	docs/conf.py
+ignore =
+	# E133: closing bracket is missing indentation
+	# D104: Missing docstring in public package
+	# D105 Missing docstring in magic method
+	# D107: Missing docstring in __init__
+	# D203: 1 blank line required before class docstring
+	# D212: Multi-line docstring summary should start at the first line
+	# D213: Multi-line docstring summary should start at the second line
+	# D301: Use r""" if any backslashes in a docstring
+	# D400: First line should end with a period
+	# D404: First word of the docstring should not be "This"
+	# D401: First line should be in imperative mood
+	# D402: First line should be in imperative mood; try rephrasing
+	# F812: list comprehension redefines ...
+	# H101: Use TODO(NAME)
+	# H202: assertRaises Exception too broad
+	# H401: docstring should not start with a space
+	# H403: multi line docstrings should end on a new line
+	# H404: multi line docstring should start without a leading new line
+	E133,D104,D105,D107,D203,D212,D213,D301,D400,D404,D401,D402,F812,H101,H202,H401,H403,H405
```

Comment:
Are all of these ignore rules really required? :)

Reply:
I'll check. It is partially cargo-culted. Recording why we're ignoring them would also be a good idea.
- [ ] Review flake8 ignored rules

Reply:
I'll check. It is partially cargo-culted. Recording why we're ignoring them would also be a good idea.
- [ ] Review flake8 ignored rules